README.md

```diff
@@ -26,339 +26,213 @@
+learning and machine learning frameworks, including TensorRT, TensorFlow, PyTorch, 
+ONNX, OpenVINO, Python and more. Triton supports inference across cloud, data center,
+edge and embedded devices on NVIDIA GPUs, x86 and ARM CPU, or AWS Inferentia. 
+Triton delivers optimized performance for many query types, including real time,
+batched, ensembles and audio/video streaming.
+
+Major features include:
+
+- [Multiple deep learning
+  framework support](https://github.com/triton-inference-server/backend).
+- [Multiple machine learning
+  framework support](https://github.com/triton-inference-server/fil_backend).
+- [Concurrent model
+  execution](docs/architecture.md#concurrent-model-execution).
+- [Dynamic batching](docs/architecture.md#models-and-schedulers).
+- Provides [Extensible
+  backend](https://github.com/triton-inference-server/backend) with a *backend
+  API*.
+- Model pipelines using
   [Ensembling](docs/architecture.md#ensemble-models) or [Business
   Logic Scripting
   (BLS)](https://github.com/triton-inference-server/python_backend#business-logic-scripting).
-  A Triton *ensemble* represents a pipeline of one or more models and
-  the connection of input and output tensors between those
-  models. *BLS* allows a pipeline along with extra business logic to
-  be represented in Python. In both cases a single inference request
-  will trigger the execution of the entire pipeline.
-
-* [HTTP/REST and GRPC inference
+- [HTTP/REST and GRPC inference
   protocols](docs/inference_protocols.md) based on the community
   developed [KServe
   protocol](https://github.com/kserve/kserve/tree/master/docs/predict-api/v2).
+- A [C API](docs/inference_protocols.md#in-process-triton-server-api) and
+  [Java API](docs/inference_protocols.md#java-bindings-for-in-process-triton-server-api) allow Triton to link directly into your application for edge and other in-process use cases.
+- [Metrics](docs/metrics.md) indicating GPU utilization, server
+  throughput, and server latency. 
+
+## Serve a Model in 3 Easy Steps
+
+```bash
+# Step 1: Create the example model repository 
+git clone -b r22.04 https://github.com/triton-inference-server/server.git
+
+cd server/docs/examples
+
+./fetch_models.sh
+
+# Step 2: Launch triton from the NGC Triton container
+docker pull nvcr.io/nvidia/tritonserver:22.04-py3
+
+docker run --gpus=1 --rm -p8000:8000 -p8001:8001 -p8002:8002 -v/full/path/to/docs/examples/model_repository:/models nvcr.io/nvidia/tritonserver:22.04-py3 tritonserver --model-repository=/models
+
+# Step 3: In a separate console, launch the image_client example from the NGC Triton SDK container
+docker pull nvcr.io/nvidia/tritonserver:22.04-py3-sdk
 
-* A [C API](docs/inference_protocols.md#in-process-triton-server-api)
-  allows Triton to be linked directly into your application for edge
-  and other in-process use cases.
+docker run -it --rm --net=host nvcr.io/nvidia/tritonserver:22.04-py3-sdk
```

Comment:
Optional, but the `docker pull ...` commands can be omitted to save a couple "quickstart" steps as they'll be pulled on `docker run` automatically if not already present

Reply:
We debated this offline, whether to call it out. But this will speed up the example, so removed.

Reply:
Done